src/main/java/com/example/onlineshop_frontend/clients/BrandClient.java

```diff
@@ -0,0 +1,16 @@
+package com.example.onlineshop_frontend.clients;
+import com.example.onlineshop_frontend.dto.Color;
+import com.example.onlineshop_frontend.dto.Brand;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.GetMapping;
+import java.util.List;
+
+@FeignClient(name = "onlineshop-api-brands", url = "http://localhost:8082/brand")
+public interface BrandClient {
+
+  @GetMapping("/getAllBrand")
```

Comment:
Same as in the back-end - we will separate the words by using "-" between them, so this endpoint should be /get-all for example.